### ISSUE TITLE
Can I get the ground truth "noun" of action recognition test set for our annotating our own label?

### ISSUE THREAD

**Issue by WikiChao** [epic-kitchens/epic-kitchens-100-annotations#24]
Hi,

We want to annotate the action recognition test set with labels associated with audio. The "noun" label will help identify what is the interacted object in the video. However, the fine-grained annotations are only provided in the train/val set. Is it possible to provide the fine-grained annotations on the test set if we are not targeting the action recognition challenge?

Thanks!

**Comment by dimadamen** [comment]
The test set is hidden behind the leaderboard for challenges and will not be publicly available or shared for any purposes due to potential leaks.
You can test your approach on the validation set.
